src/main/java/com/uipath/uipathpackage/UiPathTest.java

```diff
@@ -300,6 +310,10 @@ private void validateParameters() throws AbortException {
         if (testResultsOutputPath != null && testResultsOutputPath.toUpperCase().contains("${JENKINS_HOME}")) {
             throw new AbortException(com.uipath.uipathpackage.Messages.ValidationErrors_InvalidPath());
         }
+        
+        if (parametersFilePath != null && parametersFilePath.toUpperCase().contains("${JENKINS_HOME}")) {
```

Comment:
I see similar check for `testResultsOutputPath` as well, but what does it mean if parametersFilePath contains JENKINS_HOME? And why is that an error?

Reply:
As discussed this with Robert , he also don't know why we don't want to put in inside JENKINS_HOME , but my guess is since JENKINS_HOME points to /var/lib/jenkins and we shouldn't be placing our build related assets in that place.
also i have moved this validation from here to FormLevelValdation now.